README.md

```diff
@@ -2,6 +2,39 @@
 
 The JavaScript Ecosystem for Server-rendered Web-Applications.
 
+## Getting started
+
+## Running locally
+
+1. Fork the repository on Github.
+
+2. Clone it:
+
+        $ git clone git@github.com:your_username/hotwire.io.git 
```

Comment:
Thanks for also adding this to the README. Can you remove the `$` from the snippets so they are also copyable?

Reply:
done